### ISSUE TITLE
Thanks and requests

### ISSUE THREAD

**Issue by chaosguixiao** [Tsukina-7mochi/aseprite-scripts#17]
Hello, I am a fool who can't write scripts, I have been looking for a script to import psd files into aseprite, for this reason just stumbled across your script (export psd),  thank you for writing the script   You are a genius! After that, i searched for over a year and couldn't find the script to import psd files into aseprite. Honestly, I really hope you can help me (i will pay you).

**Comment by Tsukina-7mochi** [comment]
Thank you for the request! I'd like to make importer script, but because the complex specifications of PSD format, I'm not sure it will be useful (I mean almost all files can be unreadable.)
dacap, the developer of Aseprite also says implementing PSD related functionality is heavy work at [this topic](https://community.aseprite.org/t/layered-file-that-i-can-import-to-and-from-photoshop-gimp/743/2) .
Changing the idea, it would be hopeful to make Photoshop plugin to export .ase files.

**Comment by chaosguixiao** [comment]
Thank you so much for reply, I was naive indeed, it was very difficult to transfer all the effects from Photoshop into aseprite. Thanks for your explanation. Will this work if I just import the line and colors from the normal layer of the PSD file (without any effects) into aseprite？I'm sorry, I don't know if what I'm conveying is accurate

**Comment by Tsukina-7mochi** [comment]
Yes, what you say is true to some extent, most PhotoShop features like effects and vector layers cannot be used in Aseprite, what we can ignore it when importing PSDs. Also PSD supports features that are tricky to handle with the language used in Aseprite like multiple color depths, compression mode.
I think it seems to be more reasonable to export aseprite files from PhotoShop or use some simpler formats as an exchange format such as TIFF.

**Comment by chaosguixiao** [comment]
お返事ありがとうございます。おっしゃることはわかりましたが、どうやらPhotoshopでPNG形式の画像をレイヤごとに別に保存して、それを順次asepriteに入れていくしか方法がなさそうです。ちなみにasepriteはcspやsai2より使いにくいと思いますハハハハ。最後に詳しい説明をありがとうございました。
emmm……この後の質問があなたの気に障るかどうかわかりません。本当に失礼なことでしたら、心よりお詫び申し上げますが、お返事は差し上げません。このようなシナリオをお願いしたら（「Photoshopからasepriteファイルをエクスポートするスクリプト」または「PSDファイル(通常のレイヤコンテンツ)をasepriteにインポートするスクリプト」です。）、いくら払えばいいのでしょうか。

**Comment by Tsukina-7mochi** [comment]
(You don't need to use translator, I'm sorry if my English is strange.)

I'm grateful for your offer of support, but I'm not willing to receive financial support. It is because I don't want to have the responsibility of developing or maintaining those tools (which does not mean I don't want to make them.)
I'll notify you if I make some tool to enable import PS data to aseprite.

**Comment by Tsukina-7mochi** [comment]
P.S. [TIFF](https://www.adobe.com/creativecloud/file-types/image/raster/tiff-file.html) files can hold layer data and PS seems to support layered image exports. If aseprite can import them, does this meet your needs?

**Comment by chaosguixiao** [comment]
（Your English is very good, on the contrary I think my English may be difficult for others to understand, I think you may understand me more easily if it is Japanese. Ha ha ha)
Thank you for your reply! I have understood what you mean. I really appreciate your helping spirit.
Yes, tiff images can save layer data, but aseprite doesn't seem to be able to read images in this format (I tried to open Tiff but it didn't work).

**Comment by Tsukina-7mochi** [comment]
OK, I mean, it'll be easy to make TIFF importer instead of PSD's. So if I'm going to make a script that can read data from photoshop, I'd make a script to read TIFF.

**Comment by chaosguixiao** [comment]
Oh, I see what you mean, and I have to say you are a genius! Yeah, so you don't have to worry about removing the effects from your PSD file.
